packages/core/src/browser/style/scrollbars.css

```diff
@@ -83,7 +83,7 @@
 .ps__rail-y:hover,
 .ps__rail-y:focus {
     width: var(--theia-scrollbar-rail-width);
-    background: var(--theia-scrollbar-active-rail-color);
```

Comment:
The name of that variable sounds like it was only used to color the active rail.
Wouldn't it make sense to change that, or at least remove it completely?
Or is it still used somewhere?

Reply:
Good point. I just saw that it is used generally by all scrollbars. But here it got overwritten by the style of the perfect scrollbar itself. So `!important` is actually the only important thing here.